x-pack/plugins/observability/public/pages/slos/components/badges/slo_indicator_type_badge.tsx

```diff
@@ -28,15 +28,15 @@ export function SloIndicatorTypeBadge({ slo }: Props) {
 function toIndicatorLabel(indicatorType: SLOWithSummaryResponse['indicator']['type']) {
   switch (indicatorType) {
     case 'sli.kql.custom':
-      return i18n.translate('xpack.observability.slos.slo.indicator.customKql', {
+      return i18n.translate('xpack.observability.slo.slo.indicator.customKql', {
```

Comment:
📣 I basically renamed all translations to follow: `xpack.observability.slo.*` indicating the slo feature under observability.
Then in a following PR, I'd like to rename them to `xpack.observability.slo[.<pageName>].<componentName>.<element>`